example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -270,6 +271,7 @@
 		};
 		9740EEB61CF901F6004384FC /* Run Script */ = {
 			isa = PBXShellScriptBuildPhase;
```

Comment:
@frankus There were couple of automatic changes brought in probably by upgrading the packages.

Reply:
I don't see anything that gives me pause :)